php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -2214,16 +2214,17 @@ abstract class NDB_BVL_Instrument extends NDB_Page
      *
      * @param $commentIDs string[] A list of commentIDs to load in bulk.
      *
-     * @return \NDB_BVL_Instrument[]
```

Comment:
This type shouldn't have to change but phan isn't great at understanding generators and doesn't recognize them as `sometype[]` compatible.